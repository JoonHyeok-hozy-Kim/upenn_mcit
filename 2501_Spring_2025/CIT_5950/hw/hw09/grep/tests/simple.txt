test_files/Hello.txt Hello
test_files/Bye.txt bye
test_files/Bye.txt Goodbye
