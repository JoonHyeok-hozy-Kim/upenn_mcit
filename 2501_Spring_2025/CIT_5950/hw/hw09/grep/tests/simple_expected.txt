test_files/Hello.txt: [Hello]
Hello World!
test_files/Bye.txt: [bye]
test_files/Bye.txt: [Goodbye]
Goodbye world
Goodbye, Goodbye, Goodbye
